module/library/individualAssessments/helper.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * name : helper.js
+ * author : Aman
+ * created-date : 04-Jun-2020
+ * Description : Individual Assessment library helper functionality.
+ */
+
+// Dependencies 
+const solutionsHelper = require(MODULES_BASE_PATH + "/solutions/helper");
+const assessmentsHelper = require(MODULES_BASE_PATH + "/assessments/helper");
+
+ /**
+    * IndividualAssessmentHelper
+    * @class
+*/
+
+module.exports = class IndividualAssessmentHelper {
+
+    /**
+      * List of individual solution templates.
```

Comment:
@aman-tunerlabs - Change it to "List of individual assessment solution templates."